app/scripts/views/mixins/password-prompt-mixin.js

```diff
@@ -46,11 +49,16 @@ define(function (require, exports, module) {
     },
 
     displayPasswordWarningPrompt: function () {
+      var promptContent = TOOLTIP_MESSAGES.WARNING_PROMPT_MESSAGE;
```

Comment:
Do we want this to be added to only the warning prompt? I thought we were going to have this on all three prompts? cc/ @ryanfeeley